### PR TITLE
Adds French Localization

### PR DESCRIPTION
**Pull request by Solaarmoon**
May not be the most accurate translation since I've lost some fluency, but I feel as if this will work for most French Speakers.

### CODE REVIEW

File: iina/Base.lproj/Localizable.strings [hunk 1]
```diff
@@ -11,78 +11,78 @@
 "off" = "Off";
 
 // PlaylistViewController.swift
-"playlist.playlist_cap" = "PLAYLIST";
-"playlist.chapter_cap" = "CHAPTERS";
+"playlist.playlist_cap" = "LISTE DE LECTURE";
```
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: Apple uses the term "playlist" in the french iTunes, maybe we should stick with it ?

File: iina/Base.lproj/Localizable.strings [hunk 2]
```diff
@@ -11,78 +11,78 @@
...
+"update.new_version" = "Nouvelle version:";
+"update.dl_from_website" = "Ouverture du site Web";
+"update.dl_from_github" = "Ouverture de la page Github";
 
-"preference.title" = "Preference";
-"preference.general" = "General";
-"preference.ui" = "UI";
+"preference.title" = "Préferences";
+"preference.general" = "Général";
+"preference.ui" = "Interface d'Utilisateur";
```
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: should be "Interface utilisateur" I think ? I think we can use "UI" here ?

File: iina/Base.lproj/Localizable.strings [hunk 3]
```diff
@@ -11,78 +11,78 @@
...
-"update.check_failed" = "Failed to check updates. You are probably running a too old version. Please check the website or github.";
-"update.new_version" = "New version:";
-"update.dl_from_website" = "Go to Website";
-"update.dl_from_github" = "Go to Github";
+"update.title_update_found" = "Mise à jour Trouvée";
+"update.title_no_update" = "Aucune Mise à jour Trouvée";
+"update.title_error" = "Érreur";
+"update.update_found" = "Mise à jour trouvée! IINA est toujours en alpha, alors il est recommendé d'avoir la plus récente version. Les fichiers sont disponible sur le site web IINA ou sur GitHub:";
+"update.no_update" = "Vous avez la version la plus récente.";
+"update.check_failed" = "Echec de trouvaille de mise à jour. La version courrante est trop vieille. Les plus récent fichiers sont disponible sur le site web IINA ou sur GitHub.";
```
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: "Impossible de vérifier les mises à jour. Vous avez probablement une version trop ancienne. Veuillez consulter le site internet ou Github" ?

"Trouvaille" here is a bit weird, and there's a little mistake: "courrante" should be "courante"; you also forgot the plural in "Les plus récent fichiers sont disponible" ("Les fichiers les plus récents sont disponibles"

File: iina/Base.lproj/Localizable.strings [hunk 4]
```diff
@@ -11,78 +11,78 @@
...
-"preference.control" = "Control";
-"preference.keybindings" = "Keybindings";
-"preference.advanced" = "Advanced";
-"preference.scripts" = "Scripts";
+"preference.subtitle" = "Sous-Titres";
+"preference.network" = "Réseau";
+"preference.control" = "Controle";
+"preference.keybindings" = "Raccourcis";
+"preference.advanced" = "Paramètres Avancés";
+"preference.scripts" = "Scriptes";
```
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: "Script" is the right word for this

File: iina/Base.lproj/Localizable.strings [hunk 5]
```diff
@@ -11,78 +11,78 @@
...
 "preference.codec" = "Codec";
-"preference.subtitle" = "Subtitle";
-"preference.network" = "Network";
-"preference.control" = "Control";
-"preference.keybindings" = "Keybindings";
-"preference.advanced" = "Advanced";
-"preference.scripts" = "Scripts";
+"preference.subtitle" = "Sous-Titres";
+"preference.network" = "Réseau";
+"preference.control" = "Controle";
```
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: "Contrôle"  (you forgot an accent)

File: iina/Base.lproj/Localizable.strings [hunk 6]
```diff
@@ -11,78 +11,78 @@
...
 "menu.volume"= "Volume: %d";
-"menu.audio_delay" = "Audio Delay: %.2fs";
-"menu.sub_delay" = "Subtitie Delay: %.2fs";
-"menu.fullscreen" = "Full Screen";
-"menu.exit_fullscreen" = "Exit Full Screen";
-"menu.pause" = "Pause";
-"menu.resume" = "Resume";
+"menu.audio_delay" = "Délai Audio: %.2fs";
+"menu.sub_delay" = "Délai de Sous-Titres: %.2fs";
+"menu.fullscreen" = "Écran Plein";
```
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: should be "Plein écran"

File: iina/Base.lproj/Localizable.strings [hunk 7]
```diff
@@ -11,78 +11,78 @@
...
+"menu.exit_fullscreen" = "Fermer l'Écran Plein";
```
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: in Apple apps, it's often "Quitter le mode plein écran"
capture d ecran 2017-01-28 15 16 07

File: iina/fr.lproj/AboutWindowController.strings [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+"ZAx-ah-fbZ.title" = "mpv 0.0.0";
+
+/* Class = "NSTextFieldCell"; title = "IINA"; ObjectID = "dkA-td-UyX"; */
+"dkA-td-UyX.title" = "IINA";
+
+/* Class = "NSTextFieldCell"; title = "0.0.1"; ObjectID = "gZA-85-Zaw"; */
+"gZA-85-Zaw.title" = "0.0.1";
+
+/* Class = "NSBox"; title = "Box"; ObjectID = "igW-n0-W0W"; */
+"igW-n0-W0W.title" = "Boite";
```
Comment (superzazu) [inline iina/fr.lproj/AboutWindowController.strings]: should be "Boîte", but I think this isn't used anyways ?...

File: iina/fr.lproj/InspectorWindowController.strings [hunk 9]
```diff
@@ -0,0 +1,72 @@
+
+/* Class = "NSTextFieldCell"; title = "Format:"; ObjectID = "99G-7I-C2S"; */
+"99G-7I-C2S.title" = "Format:";
+
+/* Class = "NSBox"; title = "Audio"; ObjectID = "AI9-Ye-I3s"; */
+"AI9-Ye-I3s.title" = "Audio";
+
+/* Class = "NSBox"; title = "Video"; ObjectID = "Bbu-1w-8Vc"; */
+"Bbu-1w-8Vc.title" = "Video";
```
Comment (superzazu) [inline iina/fr.lproj/InspectorWindowController.strings]: "Vidéo"

File: iina/fr.lproj/InspectorWindowController.strings [hunk 10]
```diff
@@ -0,0 +1,72 @@
+
+/* Class = "NSTextFieldCell"; title = "Format:"; ObjectID = "99G-7I-C2S"; */
+"99G-7I-C2S.title" = "Format:";
```
Comment (superzazu) [inline iina/fr.lproj/InspectorWindowController.strings]: i'm probably nitpicking here, but there's always a space before ":" in french

File: iina/fr.lproj/QuickSettingViewController.strings [hunk 11]
```diff
@@ -0,0 +1,336 @@
...
+"qpa-hD-ImC.title" = "Cellule de Texte";
+
+/* Class = "NSTextFieldCell"; title = "Gamma:"; ObjectID = "r6x-z0-oFC"; */
+"r6x-z0-oFC.title" = "Gamma:";
+
+/* Class = "NSTextFieldCell"; title = "0s"; ObjectID = "s3F-Tl-Siy"; */
+"s3F-Tl-Siy.title" = "0s";
+
+/* Class = "NSTextFieldCell"; title = "Size:"; ObjectID = "s9l-Hb-SCk"; */
+"s9l-Hb-SCk.title" = "Grandeur:";
```
Comment (superzazu) [inline iina/fr.lproj/QuickSettingViewController.strings]: Maybe we should use "Taille" ? That seems more natural

File: iina/fr.lproj/QuickSettingViewController.strings [hunk 12]
```diff
@@ -0,0 +1,336 @@
...
+"ghy-e7-7Of.title" = "Police...";
+
+/* Class = "NSMenuItem"; title = "3"; ObjectID = "h6k-DT-fv5"; */
+"h6k-DT-fv5.title" = "3";
+
+/* Class = "NSTextFieldCell"; title = "4k"; ObjectID = "i0j-2a-vKD"; */
+"i0j-2a-vKD.title" = "4k";
+
+/* Class = "NSTextFieldCell"; title = "Equalizer"; ObjectID = "iS4-Zr-9Ig"; */
+"iS4-Zr-9Ig.title" = "Égalisateur";
```
Comment (superzazu) [inline iina/fr.lproj/QuickSettingViewController.strings]: I think the word is "égaliseur" (that's the word iTunes uses)

File: iina/fr.lproj/QuickSettingViewController.strings [hunk 13]
```diff
@@ -0,0 +1,336 @@
...
+"cpm-5o-a2c.title" = "AUDIO";
+
+/* Class = "NSTextFieldCell"; title = "8k"; ObjectID = "e4X-H4-VxS"; */
+"e4X-H4-VxS.title" = "8k";
+
+/* Class = "NSTextFieldCell"; title = "Contrast:"; ObjectID = "eDl-Si-LSa"; */
+"eDl-Si-LSa.title" = "Contraste:";
+
+/* Class = "NSTextFieldCell"; title = "Hue:"; ObjectID = "eLh-fu-pMj"; */
+"eLh-fu-pMj.title" = "Teint:";
```
Comment (superzazu) [inline iina/fr.lproj/QuickSettingViewController.strings]: "Teinte"

File: iina/fr.lproj/PrefUIViewController.strings [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+"S8H-Xr-uVa.title" = "s";
+
+/* Class = "NSButtonCell"; title = "Always float on top while playing"; ObjectID = "SFX-ct-8MB"; */
+"SFX-ct-8MB.title" = "Toujours flotter par dessus pendant la lecture";
+
+/* Class = "NSButtonCell"; title = "Use physical resolution on retina screens"; ObjectID = "fsj-YE-OB7"; */
+"fsj-YE-OB7.title" = "Utiliser la résolution physique sur les écran retina";
+
+/* Class = "NSTextFieldCell"; title = "On Screen Controller:"; ObjectID = "g06-Ja-Ot8"; */
+"g06-Ja-Ot8.title" = "Controlleur Sur l'Écran:";
```
Comment (superzazu) [inline iina/fr.lproj/PrefUIViewController.strings]: "Contrôleur sur l'écran"

File: iina/fr.lproj/PrefSubViewController.strings [hunk 15]
```diff
@@ -0,0 +1,123 @@
...
+"PZ2-pZ-hHa.title" = "Autre:";
+
+/* Class = "NSMenuItem"; title = "All subtitles in the same folder"; ObjectID = "Qcl-0x-3hh"; */
+"Qcl-0x-3hh.title" = "Tous les Sous-Titres dans le même dossier";
+
+/* Class = "NSTextFieldCell"; title = "Font:"; ObjectID = "RiL-Rv-eke"; */
+"RiL-Rv-eke.title" = "Police:";
+
+/* Class = "NSTextFieldCell"; title = "Background:"; ObjectID = "SLQ-wt-k6Y"; */
+"SLQ-wt-k6Y.title" = "Fond d'écran:";
```
Comment (superzazu) [inline iina/fr.lproj/PrefSubViewController.strings]: "Fond d'écran" is a wallpaper; what we're looking for here is "arrière-plan" I think :)

File: iina/fr.lproj/PrefControlViewController.strings [hunk 16]
```diff
@@ -0,0 +1,66 @@
...
+"1yb-m4-1sK.title" = "Scroll horizontally to:";
+
+/* Class = "NSMenuItem"; title = "Previous / Next Media"; ObjectID = "2Ig-fw-UvW"; */
+"2Ig-fw-UvW.title" = "Média Précedent / Prochain";
+
+/* Class = "NSMenuItem"; title = "Toggle fullscreen"; ObjectID = "6F4-gm-oBg"; */
+"6F4-gm-oBg.title" = "Activer/Désactivr l'Écran Plein";
+
+/* Class = "NSTextFieldCell"; title = "Exact seek is more precise but can cause lag and high CPU usage."; ObjectID = "6FZ-Qm-0k0"; */
+"6FZ-Qm-0k0.title" = "La Recherche Exacte est plus précise, mais peut entraîner du lag et un utilisation de processeur élevé.";
```
Comment (superzazu) [inline iina/fr.lproj/PrefControlViewController.strings]: no need to capitalize "Recherche Exacte", and it should be : "une utilisation de processeur élevée" (le terme "utilisation" est féminin).

File: iina/fr.lproj/PrefCodecViewController.strings [hunk 17]
```diff
@@ -0,0 +1,24 @@
...
+"m5P-5f-5uo.title" = "Video:";
+
+/* Class = "NSTextFieldCell"; title = "Audio:"; ObjectID = "sV5-dL-MUt"; */
+"sV5-dL-MUt.title" = "Audio:";
+
+/* Class = "NSTextFieldCell"; title = "Number of threads:"; ObjectID = "tO4-pw-2W9"; */
+"tO4-pw-2W9.title" = "Nombres de fil:";
+
+/* Class = "NSTextFieldCell"; title = "Prefered language:"; ObjectID = "wjt-O6-KbC"; */
+"wjt-O6-KbC.title" = "Langue Préferrée:";
```
Comment (superzazu) [inline iina/fr.lproj/PrefCodecViewController.strings]: "Langue préférée"

File: iina/fr.lproj/PrefCodecViewController.strings [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+"bxW-np-SGm.title" = "Défaut: 0 (Auto)";
+
+/* Class = "NSTextFieldCell"; title = "Video:"; ObjectID = "m5P-5f-5uo"; */
+"m5P-5f-5uo.title" = "Video:";
+
+/* Class = "NSTextFieldCell"; title = "Audio:"; ObjectID = "sV5-dL-MUt"; */
+"sV5-dL-MUt.title" = "Audio:";
+
+/* Class = "NSTextFieldCell"; title = "Number of threads:"; ObjectID = "tO4-pw-2W9"; */
+"tO4-pw-2W9.title" = "Nombres de fil:";
```
Comment (superzazu) [inline iina/fr.lproj/PrefCodecViewController.strings]: Nombre de processus ?

**Comment by superzazu** [comment]
I allowed myself to add comments on your translation (I'm french).
Well, funny thing, I also made [a french translation](https://github.com/superzazu/iina/commit/c8a1bbf55d5d0d0a87578027c585493bbc4321ba) during the week.

**Comment by lhc70000** [comment]
@Solaarmoon  Any update?

**Comment by metacosm** [comment]
Seems to me that @superzazu's translation is better overall, both in terms of French and files. I've created a PR for @superzazu's work to both update his branch with latest changes from master and add missing translations for #291

**Comment by Solaarmoon** [comment]
@metacosm I'll agree. I never use my OS in French, so I don't have knowledge of the common language used. Good Job @superzazu !

**Comment by metacosm** [comment]
@Solaarmoon thanks for getting the ball rolling on this! 👍

**Comment by superzazu** [comment]
Thank you @metacosm for your PR, and thank you @Solaarmoon (again, sorry for not saying I was working on this) 👍 

I'll open another PR then :)

**Comment by metacosm** [comment]
Please close this PR and replace by #298

File: iina/Base.lproj/Localizable.strings (hunk 3 above)
Comment (Aureleme) [inline iina/Base.lproj/Localizable.strings]: "Érreur" don't have a "É", just "Erreur" or "erreur".

File: iina/Base.lproj/Localizable.strings (hunk 4 above)
Comment (Aureleme) [inline iina/Base.lproj/Localizable.strings]: "Controle" has a "ô", "Contrôle". It's perhaps a plural ? "Contrôles"

File: iina/fr.lproj/AboutWindowController.strings (hunk 8 above)
Comment (Aureleme) [inline iina/fr.lproj/AboutWindowController.strings]: Don't worry about the "î", a simple "i" is fine :)

File: iina/Base.lproj/Localizable.strings (hunk 3 above)
Comment (superzazu) [inline iina/Base.lproj/Localizable.strings]: @Aureleme you're in the wrong PR, this one is going to be closed, see here => https://github.com/lhc70000/iina/pull/298

**Pull request closed without merging**